Server/Models/AppDbContext.cs

```diff
@@ -44,6 +50,13 @@ protected override void OnModelCreating(ModelBuilder builder)
             builder.Entity<Player>()
                 .Property(s => s.Rating12v12)
                 .HasDefaultValue(Player.DefaultRating);
+
+
+            builder.Entity<Player>()
+                .OwnsMany(p => p.PlayerOverthrowRating, a =>
```

Comment:
Originally by owned entity I have meant `OwnsOne` (with all maps being class properties, with some dynamic lookup helpers). I don't see much point in `OwnsMany`, it doesn't seem to give anything regular relationships doesn't